src/Interfaces/ISyncSwapPool.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+/// @title Interface for SyncSwapV2 Vault
+/// @author LI.FI (https://li.fi)
+/// @custom:version 1.0.0
+/// @notice This interface is used to interact with the SyncSwapV2 Vault
```

Comment:
typo? Should reference ISyncSwapPool instead no?

Reply:
fixed